samples/snippets/storage_download_blob_as_chunks_concurrently.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def download_blob_as_chunks_concurrently(bucket_name, blob_name, filename, processes=8):
```

Comment:
Let's just call this "download chunks concurrently" to have parity with the underlying function. Please also update the file name if you don't mind. Thanks!